nussknacker-dist/src/universal/conf/dev-application.conf

```diff
@@ -165,7 +166,8 @@ scenarioTypes {
   }
   "streaming-lite-k8s": {
     deploymentConfig {
-      type: "streaming-lite-k8s"
+      type: "lite-k8s"
+      mode: "streaming"
       //FIXME: simplify, make it more human-friendly...
```

Comment:
maybe just extract common config ?

Reply:
This comment is copy-paste from below. I would like to avoid duplication but IMO the way to do it is to make deployment manager <> scenario type relation one to many instead one to one like it is now. I left somewhere TODO for that